### PR TITLE
Jails/influxdb

### PR DESCRIPTION
**Pull request by colemamd**
Cleanup & bug fixes

### CODE REVIEW

**Comment by colemamd** [comment]
Just realized I didn't clean everything up before I made the PR. I'll update soon

**Comment by colemamd** [comment]
So I definitely had the whole persistence screwed up. This should fix it. The database is fully stored in /config now.

**Comment by Ornias1993** [comment]
@colemamd Thanks for your work, but mind squashing it? It now includes all sorts of previous commits in your PR. Which it shouldn't....

File: jails/influxdb/install.sh [hunk 1]
```diff
@@ -7,30 +7,30 @@
...
 JAIL_IP="$(sed 's|\(.*\)/.*|\1|' <<<"${influxdb_ip4_addr}" )"
 INCLUDES_PATH="${SCRIPT_DIR}/jails/influxdb/includes"
 DATABASE=${influxdb_database}
 
-# Enable influxdb
+# Mount and configure proper configuration location
+cp -rf "${INCLUDES_PATH}/influxd.conf" "/mnt/${global_dataset_config}/${JAIL_NAME}/influxd.conf"
+iocage exec "${JAIL_NAME}" mkdir -p /config/db/data /config/db/meta /config/db/wal
+iocage exec "${JAIL_NAME}" chown -R influxd:influxd /config/db
+iocage exec "${JAIL_NAME}" sysrc influxd_conf="/config/influxd.conf"
```
Comment (Ornias1993) [inline jails/influxdb/install.sh:20]: Ahh this is a nice solution!

Comment (colemamd) [reply jails/influxdb/install.sh:20]: Took me quite a while to figure that out. lol

Comment (colemamd) [reply jails/influxdb/install.sh:20]: The sysrc config function isn't documented anywhere, except as a comment in the daemon file.

**Comment by Ornias1993** [comment]
Do you know how to rebase and/or squash changes?
if not, maybe try to find a good guide ;)

**Comment by colemamd** [comment]
Haha no! Been trying to find a good guide. I've screwed up my repo (I think) and trying to fix it now.

**Comment by Ornias1993** [comment]
I manually rebased it and merged it into dev for you :)

**Comment by colemamd** [comment]
Ahh, thanks. I just went through and manually rebased and squashed everything and did a force push... But now I know the process, and I'll make sure and squash prior to another PR. thanks!

**Pull request closed without merging**